.github/workflows/lint.pr.yml

```diff
@@ -0,0 +1,18 @@
+name: Lint
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: current
```

Comment:
Just curious, what's the reason for choosing `current` node version here? In the [actions/setup-node](https://github.com/actions/setup-node#usage) readme they say `"it is recommended to always specify Node.js version and don't rely on the system one."`

Reply:
I see. Just to understand what could happen in the future with this. Can using “current” mean that if there’s a breaking change in node that affects our code in warp drive, the job will fail forcing us to either update the scripts or change the node version to what we’ve supported so far?

Reply:
Yes, but this is a good thing. We *want* our tests to fail when a new version of Node would cause something to break - that way we find out about it before our users do!

Reply:
Correct me if I'm wrong but I thought that specifying node version within a module is only relevant to that module and not other apps that use that module. Then if we expect the module to work on Node 18 per today and a github action fails in a year from now because it's using node 20, then my understanding was that we just affect ourselves.

Reply:
That sounds right?

Reply:
😄 Then I didn't get what you meant by that comment:
> We want our tests to fail when a new version of Node would cause something to break - that way we find out about it before our users do